src/App.js

```diff
@@ -1,27 +1,39 @@
-import logo from './logo.svg';
-import './App.css';
+// import './App.css';
+import Wrapper from './components/Wrapper';
+import Display from './components/Display';
+import Button from './components/Button';
+import ButtonPanel from './components/ButtonPanel';
+
+const btnValues = [
+  ['C', '+-', '%', '/'],
+  [7, 8, 9, 'X'],
+  [4, 5, 6, '-'],
+  [1, 2, 3, '+'],
+  [0, '.', '='],
+];
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Wrapper>
+        <Display value="0" />
+        <ButtonPanel>
+          {btnValues.flat().map((btn) => (
+            <Button
+              key={btn}
+              className={btn === '=' ? 'equals' : ''}
+              value={String(btn)}
+              onClick={() => {
+                console.log(`${btn} clicked!`);
```

Comment:
- [ ] Please avoid the use of console.log in code being reviewed, you are free to use it while developing but it should not be present in the pull request, plus this is giving warning errors when running your app, please remove it for something else 👍